src/options/index.ts

```diff
@@ -69,6 +79,42 @@ export const options = {
 			}
 		]
 	},
+	closingBracketPosition: {
+		since: '1.2.1',
+		category: CATEGORY_PUG,
+		type: 'choice',
+		default: 'new-line',
+		description: 'Determines position of closing bracket which wraps attributes',
+		choices: [
+			{
+				value: 'new-line',
+				description:
+					`
+					Example:
```

Comment:
Before the `Example`, please add a descriptive text for both possible values

PS: same sentences as in README.md